libr/cons/cons.c

```diff
@@ -1268,7 +1268,7 @@ R_API int r_cons_get_column(void) {
 
 /* final entrypoint for adding stuff in the buffer screen */
 R_API int r_cons_write(const char *str, int len) {
-	if (R_STR_ISEMPTY (str) || len < 0) {
+	if (len < 0) {
```

Comment:
can you add ` r_return_val_if_fail (str && len <1);`

and make ` if (len == 0) return 0;`?

Reply:
Yes I will do it this today.

Reply:
I already did that i. Master. Do you want to extend alias files to support null bytes too? I can explain better about what are alias files and how to handle them in discord